src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -464,6 +465,13 @@ private void InitializeSuffixes()
             AddSuffix("MESSAGES", new NoArgsSuffix<MessageQueueStructure>(() => GetMessages()));
         }
 
+        public global::Part GetControlPart()
+        {
+            global::Part res = Vessel.GetReferenceTransformPart(); //this can actually be null if "control from here" has never been used
```

Comment:
Of course, you may change the comment as you see fit.
The craft that gave me the null reference had its probe core as the root part. Maybe if the root isn't a control module (or maybe a docking port), it always switches control to a core or a capsule, if there's one (and crewed modules may even override probe cores in the root)
Uncontrollable root part as the control reference is probably left only for uncontrollable debris